core/src/main/scala/chisel3/reflect/DataMirror.scala

```diff
@@ -330,6 +330,13 @@ object DataMirror {
     }.map(_._1).flatten
   }
 
+  /** Check if the given Data is fully aligned w.r.t base.  Chisel version of FIRRTL "passive".
+    *
```

Comment:
should we add a note that a thing is always aligned to itself, even if it is `Input`, or `Flipped`, or whatever at the base?